roles/openshift_cluster_metrics_hawkular/tasks/main.yaml

```diff
@@ -0,0 +1,56 @@
+---
+
+- name: Copy Configuration to temporary conf
+  command: >
+    cp {{ openshift.common.config_base }}/master/admin.kubeconfig {{hawkular_tmp_conf}}
+  changed_when: false
+
+- name: Install cluster metrics templates
+  copy:
+    src: cluster-metrics
+    dest: "{{ openshift.common.config_base }}/"
+
+- name: Create metrics-deployer Service Account
```

Comment:
After discussing this a bit, @abutcher and I believe that this is something that we should handle as a later refactoring (considering the changes needed to the way we are handling service accounts, secrets and permissions).